Lib/dis.py

```diff
@@ -295,12 +315,48 @@ class Instruction(_Instruction):
          argval - resolved arg value (if known), otherwise same as arg
          argrepr - human readable description of operation argument
          offset - start index of operation within bytecode sequence
+         start_offset - start index of operation within bytecode sequence including extended args if present.
+                        Otherwise equal to Instruction.offset
          starts_line - line started by this opcode (if any), otherwise None
          is_jump_target - True if other code jumps to here, otherwise False
          positions - Optional dis.Positions object holding the span of source code
                      covered by this instruction
     """
 
+    @property
+    def oparg(self):
+        """Alias for Instruction.arg"""
+        return self.arg
+
+    @property
+    def baseopcode(self):
+        """numeric code for the base operation if operation is specialized.
+        Otherwise equal to Instruction.opcode
```

Comment:
```suggestion
        """Numeric code for the base operation if operation is specialized.

        Otherwise equal to Instruction.opcode.
```

Please apply similar markup/punctuation improvements to the remaining docstrings.